changelog/pending/20250517--cli--add-postgresql-diy-backend.yaml

```diff
@@ -0,0 +1,11 @@
+changes:
+- type: enhancement
+  scope: cli
+  description: |
+    Add PostgreSQL support as a backend for Pulumi. 
+    Users can now store Pulumi state in a PostgreSQL database using the `postgres://` URL scheme.
+    Example: `pulumi login postgres://username:password@hostname:5432/database?sslmode=disable`
```

Comment:
Hrm, `sslmode=disable` is probably not what we should recommend, and thus put in the changelog?